HaikuPorter/ShellScriptlets.py

```diff
@@ -676,8 +676,18 @@ def getShellVariableSetters(shellVariables):
 
 	local objcopy=$(getTargetArchitectureCommand objcopy)
 	local strip=$(getTargetArchitectureCommand strip)
+	local xres=$(getTargetArchitectureCommand xres)
+	local tmpfile="tmp.rsrc"
```

Comment:
It seems a bad idea to always use the same filename, I would recommend using the mktemp command to generate a random (unique) file for you instead. This could avoid problems in parallel builds.
Aternatively, you can use the original filename and append .rsrc.